### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by Ademola-coding**
- The content of the email field has to be in lower case.
   - You should implement the following interactions:
   - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

I have implemented all this in my project.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -220,3 +220,18 @@ for (let i = 0; i < projbtnD.length; i += 1) {
...
+const errMsg = document.querySelector('form > .errmsg');
+
+subBtn.addEventListener('click', (event) => {
+  const { value } = emails;
+  if (value !== value.toLowerCase()) {
+    event.preventDefault();
+
+    errMsg.textContent = 'kindly enter lowerCase character in the email field!';
+  }
+});
```
Comment (Ugboaja-Uchechi) [inline app.js:237]: - [ ] Kindly make sure you validate your form to check that the email is written in all lower case. Right now it doesn't work.

[Video](https://www.awesomescreenshot.com/video/10111970?key=f229dca057438d5c0b2a9d5b35126182)

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Ademola-coding , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (Ademola-coding) [reply app.js]: I forget the desktop validation!

Comment (Ademola-coding) [reply app.js]: Done

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Ademola-coding  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**